docs/faq.md

```diff
@@ -34,7 +34,7 @@ A DCB-compliant Event Store offers features that are not typically found in trad
 
 However, we’re experimenting with an adapter layer that allows traditional Event Stores to support DCB features by incorporating strategies like pessimistic locking to enforce consistency where needed. Stay tuned.
 
-## Why do you want to kill aggregates?
+## Why do you want to kill aggregates? 😉
```

Comment:
replace with a friendly phrase (including a smiley) in the response

Reply:
I rephrased the response to be more clear – but the smiley didn't really fit in. Do you agree to the current version?